flake.nix

```diff
@@ -134,14 +137,14 @@
             ];
           };
 
+          webui = pkgs.callPackage ./webui { };
           homestakeros = pkgs.mkYarnPackage {
             pname = "homestakeros";
             version = "0.0.1";
 
             src = ./.;
             packageJSON = ./package.json;
             yarnLock = ./yarn.lock;
-            yarnNix = ./yarn.nix;
           };
```

Comment:
Here, we should patch the code after `unpackPhase`, in a phase called `https://ryantm.github.io/nixpkgs/stdenv/stdenv/#var-stdenv-postUnpack` or `postPatch`.

So:

```
postUnpack = ''
  substituteInPlace ./app.js \
    --replace webui/dist ${webui}/dist
''
```

For example, see: https://github.com/jhvst/nix-config/blob/8c5227fb6e71d4efae8fb60e9a601c5f4aa1d58e/packages/vkroots/default.nix#L22

This way, the webui will become a dependency of the homestakeros package, and the dist will point to nix store path of the webui.

If we do this, then the schema fill will probably have to be included in git, unless it can be called from the build script of the webui.